BusBoard.ConsoleApp/Program.cs

```diff
@@ -25,22 +25,17 @@ static void Main(string[] args)
 
             var coordinate = postcodeApiHelper.GetPostcodeCoordinates(input);
             var stopPoints = tflApiHelper.GetStopPoints(coordinate);
-            if (stopPoints == null)
-            {
-                Console.WriteLine("No nearby stops found for that postcode!");
-            }
+            var output = new List<string>();
 
-            else
+            if (stopPoints != null)
             {
-                var buses = new List<BusEntry>();
-                foreach (var stopPoint in stopPoints.Take(2))
-                {
-                    buses.AddRange(tflApiHelper.GetBuses(stopPoint.naptanId));
-                }
-                var ordered = buses.OrderBy(bus => bus.expectedArrival);
-                var topFive = ordered.Take(5).Select(bus => bus.GetBusData());
-                Console.WriteLine(string.Join("\n", topFive));
+                output = stopPoints
+                    .Take(2)
+                    .Select(stop => tflApiHelper.GetTopFiveBuses(stop.naptanId))
+                    .ToList();
```

Comment:
Do you need it to be a list? 🙂

Reply:
Only needed to be a list as "var output = new list" made the type of the output variable a list